uvicorn/importer.py

```diff
@@ -1,11 +1,12 @@
 import importlib
+from typing import Any
 
 
 class ImportFromStringError(Exception):
     pass
 
 
-def import_from_string(import_str):
+def import_from_string(import_str: Any) -> Any:
```

Comment:
```suggestion
def import_from_string(import_str: Union[str, ModuleType]) -> ModuleType:
```

I've not tested it but it is what I would do, does this work ? Just asking if we can avoid `Any` here

Reply:
Oh! That's so cool. I didn't know about ModuleType... I'm going to try it later. Thank you! :)

Reply:
It works. Applied.